File: src/test/java/io/plagov/advent/aoc2016/Day05Test.java
package io.plagov.advent.aoc2016;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class Day05Test {

    @Test
    void solveFirst() {
        var day5 = new Day05();
        var password = day5.partOne("reyedfim");

        Assertions.assertEquals("f97c354d", password);
    }

    @Test
    void solveSecond() {
        var day5 = new Day05();
        var password = day5.partTwo("reyedfim");

        Assertions.assertEquals("863dde27", password);
    }
}

File: src/test/kotlin/io/plagov/advent/aoc2024/Day05Test.kt
package io.plagov.advent.aoc2024

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

import util.readInputFileAsString

class Day05Test {

  private val input = readInputFileAsString("2024/day05.txt")

  private val day5 = Day05()

  @Test
  fun partOne() {
    Assertions.assertEquals(5452, day5.partOne(input))
  }
}
